Cargo.toml

```
@@ -0,0 +1,7 @@
+[workspace]
+
+members = [
+    "internal/policies/mount/adsys_mount",
+]
+
+exclude = [ "vendor_rust/" ]
```

I would suggest enabling LTO for the release profile by default so that the binary could be even more optimized and smaller in size.

```suggestion
exclude = [ "vendor_rust/" ]

[profile.release]
lto = "fat"
```